docs/reference/buildx_dial-stdio.md

```diff
@@ -0,0 +1,15 @@
+# docker buildx dial-stdio
+
+<!---MARKER_GEN_START-->
+Dial stdio
+
+### Options
+
+| Name         | Type     | Default | Description                                      |
+|:-------------|:---------|:--------|:-------------------------------------------------|
+| `--builder`  | `string` |         | Override the configured builder instance         |
+| `--platform` | `string` |         | Target platform: this is used for node selection |
+
+
+<!---MARKER_GEN_END-->
+
```

Comment:
Let's add a bit more description/usage here. It isn't entirely obvious for new user what "dial stdio" means.

Reply:
Added extra docs.

Reply:
Not sure what changed for this

Reply:
🤔 
I wrote stuff... apparently I did not commit it somehow.

Reply:
Pushed up again.

Reply:
Oh I know what happened... I did docs the wrong way.